Dockerfile

```diff
@@ -13,7 +13,7 @@ RUN esc -o cmd/gangway/bindata.go templates/
 
 RUN CGO_ENABLED=0 GOOS=linux go install -ldflags="-w -s" -v github.com/heptiolabs/gangway/...
 
-FROM alpine:latest
-RUN apk --no-cache add ca-certificates
+FROM amd64/debian:latest
```

Comment:
Can this be debian-slim? Suggest pinning to a specific version. For example, `10.3-slim`.

Reply:
+1 pinning a specific version

Reply:
Updated with debian:9.12-slim as base image